src/main.dart

```diff
@@ -1,5 +1,183 @@
-import 'core/note_app.dart';
+import 'dart:io';
 
+class Note {
+  late String title;
+  late String content;
+
+  Note({
+    required this.title,
+    required this.content,
+  });
+}
+
+//Manage operation of program (create,edit,delete,search,display).
+class NoteManager{
+  List <Note> notes=[];
+
+
+//Creates a new note and adds it to the list of notes.
+ void createNote(){
+  String title,content;
+  print("Enter the note title: ");
+  title=stdin.readLineSync() ?? "None";
```

Comment:
It doesn't make sense for a user to create a note without providing a name.
![Screen Shot 1445-07-21 at 4 26 49 PM](https://github.com/GDSC-IAU/Selection-Project/assets/86604450/e057eb4d-5f57-4a53-b250-c2a2284cf833)